packages/jest-cli/src/reporters/notify_reporter.js

```diff
@@ -31,21 +32,37 @@ export default class NotifyReporter extends BaseReporter {
     super();
     this._globalConfig = globalConfig;
     this._startRun = startRun;
+    this._previousSuccess = false;
+    this._firstRun = true;
   }
 
   onRunComplete(contexts: Set<Context>, result: AggregatedResult): void {
     const success =
       result.numFailedTests === 0 && result.numRuntimeErrorTestSuites === 0;
 
-    if (success) {
+    const notifyMode = this._globalConfig.notifyMode;
```

Comment:
Not sure what else I'm missing but when I log globalConfig I see that notify is set to true but notifyMode is undefined. However when I go into https://github.com/facebook/jest/blob/v22.0.0/packages/jest-cli/src/cli/index.js#L72
I see that notifyMode is set to the value I pass in.

Reply:
I" think you might be hit by https://github.com/facebook/jest/pull/4639/files#r143555141

Reply:
That was it 🥇 still have to work on the change option though. It seems like a new instance of NotifyReporter is created every time `jest --watch` reruns. I'll look into that.